pages/cookbook/nfts.mdx

```diff
@@ -2,8 +2,215 @@
 
 import { Callout } from 'nextra/components'
 
-<Callout emoji="🚨">
+This page lists common examples of working with [NFTs](https://docs.ton.org/develop/dapps/asset-processing/nfts).
 
-  This page is a stub. [Contributions are welcome!](https://github.com/tact-lang/tact-docs/issues)
+## Accepting NFT ownership assignment
 
+Notification message of assigned NFT ownership has the following structure:
+
+```tact
+message(0x05138d91) NFTOwnershipAssigned {
+    previousOwner: Address;
+    forwardPayload: Slice as remaining;
+}
+```
+
+Use [receiver](/book/receive) function to accept notification message.
+
+<Callout type="warning" emoji="⚠️">
+
+  Sender of notification must be validated!
+
+</Callout>
+
+Validation can be done using [`StateInit{:tact}`](/book/expressions#initof) and derived address of the NFT item.
+Initial data layout of the NFT item is shown below, but sometimes it may differ.
```

Comment:
same suggestion as for https://github.com/tact-lang/tact-docs/pull/380#discussion_r1760022463.

In real cases the validation will usually happen by address that was stored in contract's data previously, not by calculating it from stateinit (which may differ from the standard one)

Reply:
resolved